vividus-extension-selenium/src/main/java/org/vividus/selenium/screenshot/ScreenshotTaker.java

```diff
@@ -30,4 +30,6 @@
     Path takeScreenshotAsFile(String screenshotName) throws IOException;
 
     Path takeScreenshot(Path screenshotFilePath) throws IOException;
+
+    void takeScreenshotInAllure(String screenshotName);
```

Comment:
why is new method needed? why can't we use `Optional<Screenshot> takeScreenshot(String screenshotName);`?